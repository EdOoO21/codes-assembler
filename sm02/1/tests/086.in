-937 139 715 -350 521 -976 574 -858 449 28 516 293 -379 682 477 673 -225 -790 808 769 35 -804 678 -163 -58 -1000 189 571 555 -565 -574 -250 -873 603 812 785 -648 -189 -417 831 -828 -31 -193 -822 -399 -18 -717 932 665 11 -240 -124 -125 559 -223 410 -589 -584 461 729 -915 624 -185 -474 -350 72 513 327 -236 -251 541 383 -206 255 -158 -109 211 592 -480 -663 246 441 789 598 10 368 84 846 468 642 -76 -137 -887 948 -792 519 -972 883 -633
