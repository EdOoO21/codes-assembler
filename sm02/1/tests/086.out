-633
883
-972
519
-792
948
-887
-137
-76
642
468
846
84
368
10
598
789
441
246
-663
-480
592
211
-109
-158
255
-206
383
541
-251
-236
327
513
72
-350
-474
-185
624
-915
729
461
-584
-589
410
-223
559
-125
-124
-240
11
665
932
-717
-18
-399
-822
-193
-31
-828
831
-417
-189
-648
785
812
603
-873
-250
-574
-565
555
571
189
-1000
-58
-163
678
-804
35
769
808
-790
-225
673
477
682
-379
293
516
28
449
-858
574
-976
521
-350
715
139
-937
